src/modules/playRoom/components/ResultRoll.js

```diff
@@ -1,39 +1,48 @@
-import { MASCOTS } from '../../../core/constants/mascots';
-import imgTom from '../../../assets/images/image-pick/tom.png';
-import imgCa from '../../../assets/images/image-pick/ca.png';
-import imgCua from '../../../assets/images/image-pick/cua.png';
+import { useState, useEffect } from 'react';
+import { socket } from '../../../core/services/socket';
+import imgResultWaiting from '../../../assets/images/resultWaiting.jpg';
+import convertResult from '../../../core/helper/convertResult';
 
 const ResultRoll = () => {
-  // Fake data to display
-  const imagesResultRoll = [
-    { 
-      id: 1,
-      src: imgTom,
-      value: MASCOTS.TOM,
-    },
-    { 
-      id: 2,
-      src: imgCa,
-      value: MASCOTS.CA,
-    },
-    { 
-      id: 3,
-      src: imgCua,
-      value: MASCOTS.CUA,
-    },
-  ];
+  const [results, setResults] = useState(null);
+  const [showResult, setShowResult] = useState(false);
+
+  useEffect(() => {
+    socket.on('return-result-to-player', (returnResult) => {
+      let resultConveted = returnResult.rollResult.map((item) => {
```

Comment:
Check `returnResult.rollResult.map` carefully otherwise you can experience null pointer exception